File: abc/abc176/abc176_b.cs
// https://atcoder.jp/contests/abc176/submissions/16152795

using System;
using System.Linq;

class Program
{
    static void Main()
    {
        var N = Console.ReadLine().ToCharArray().Select(c => (int)(c - '0')).ToArray();
        var sum = 0;
        for (int i = 0; i < N.Length; i++)
        {
            sum += N[i];
        }
        Console.WriteLine(sum % 9 == 0 ? "Yes" : "No");
    }
}

File: abc/abc176/abc176_b.fs
// https://atcoder.jp/contests/abc176/submissions/16152021

stdin.ReadLine().ToCharArray()
|> Array.sumBy (fun x -> (int) x - (int) '0')
|> fun x ->
    if x % 9 = 0 then "Yes"
    else "No"
|> stdout.WriteLine
